frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -27,21 +29,52 @@ export enum ClusterUpdateStatus {
 
 export const clusterVersionReference = referenceForModel(ClusterVersionModel);
 
-export const getAvailableClusterUpdates = (cv: ClusterVersionKind): ClusterUpdate[] => {
-  return _.get(cv, 'status.availableUpdates', []);
+const getAvailableClusterUpdates = (cv: ClusterVersionKind): Release[] => {
+  return cv.status?.availableUpdates || [];
```

Comment:
![image](https://user-images.githubusercontent.com/8126518/167034232-5c251163-b49b-4929-9001-6a3a04d6d44e.png)

I got this the last time I was testing with the cluster bot ... but on refresh it goes away. I think you need to address this -- maybe only for fresh clusters? Not sure how to cause it to happen multiple times.
```suggestion
  return cv?.status?.availableUpdates || [];
```

Reply:
Good catch.  [`status` is indeed optional](https://github.com/openshift/api/blob/6e7069f2fba25eeeaec78b1660e9415ca4898ad5/config/v1/types_cluster_version.go#L28).